.github/workflows/pytest.yml

```diff
@@ -35,9 +35,28 @@ jobs:
           python -m pip install .[test]
 
       - name: Run tests
+        id: tests
         run: make test
+        continue-on-error: true
 
-      - name: Analysing the code with pylint
-        if: ${{ matrix.python-version }} == '3.10'
+      - name: Pylint analysis
+        id: pylint_analysis
         run: |
-          pylint --fail-under=9 $(git ls-files '*.py')
\ No newline at end of file
+          python -m pylint --fail-under=10 $(git ls-files '*.py')
```

Comment:
Any PR CI runs are now going to fail if Pylint finds anything it is not happy about.

Reply:
This is quite strict, meaning the code quality according to Pylint needs to get a score of 10/10. If you say it is a bit too strict, I'm happy for this to go down to 9/10. Your decision :)

Reply:
For me, 10 is fine (indeed very instructive).